### PR TITLE
Added accepted programs tags

### PR DESCRIPTION
**Pull request by thumn**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR
`StoreCard` and `StoreDetailsScreen` now display:
- Ward number
- Distance from your current location
- Tags for accepted programs

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

### Online sources
[`react-native-paper` Chip](https://callstack.github.io/react-native-paper/chip.html)

[//]: # "Optional - copy links to any tutorial or documentation that was useful to you when working on this PR"

### Related PRs
#96 

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## How to review
Navigate to different store cards through the map, in the `StoreList` screen, and through the search feature. Make sure tags are showing up correctly in the `StoreCard` and that everything looks visually centered in the `StoreDetailsScreen`.

For example, it looks like Grubbs is the only store that has the WIC tag. Nam's is currently the only store that has the Healthy Rewards tag.

[//]: # "The order in which to review files and what to expect when testing locally"

## Next steps
I could not for the life of me figure out how to stop the tag descriptions from overflowing the page on the `StoreDetailsScreen` even with the use of `flexWrap: wrap`. I'm submitting this PR anyway because we have bug squashing tomorrow, and all that's left is really a bunch of visual bugs. The main functionality of the tags is finished, so if we need that by tomorrow, here it is!

Also, the tags should be left-justified in `StoreDetailsScreen`, but when I tried to left justify everything, the widths would expand and it would look pretty gross. The widths of the chips only look right on this screen because of `alignItems: 'center'` for some reason. Any tips would be APPRECIATED! :'(

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

## Tests Performed, Edge Cases
Tested tag overflow by making the widths comically large. Tested with edge case stores like Grubbs and Nam's.

[//]: # "Hopefully we will add a testing suite/CI soon, but until then note down the steps you took to test locally"

### Screenshots
IMG_2714
IMG_2715

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @tommypoa  @wangannie

[//]: # "This tags in both Annies as a default. Feel free to change, or add on anyone who you should be in on the conversation."

### CODE REVIEW

File: screens/map/StoreDetailsScreen.js [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+                    )}
+                    style={styles.chip}>
+                    EBT
+                  </Chip>
+                )}
+                {wic && (
+                  <Chip
+                    icon={() => (
+                      <FontAwesome5
+                        name="heart"
```
Comment (wangannie) [inline screens/map/StoreDetailsScreen.js]: Add prop `solid` to make it the filled icon

**Comment by wangannie** [comment]
Notes on what I changed:
* Modified sizing of the chips to better match the Figma (smaller cards, smaller font, text color, Poppins font)
* As a temporary fix for wrapping issues, I made the description text truncate into ellipses
* Some temporary copy changes ex 'Accepts Healthy Rewards' instead of 'Participates in Healthy Rewards' because long text is HARD AND WRAPPING IS SO ANNOYING AH
* Left-aligned chips in StoreDetails

File: components/store/StoreCard.js [hunk 2]
```diff
@@ -55,89 +61,153 @@ export default function StoreCard({
...
+                  size={10}
+                  color={Colors.darkerGreen}
+                  style={{ marginTop: -10 }}
+                />
+              )}
+              textStyle={styles.chipText}
+              style={styles.chip}>
+              <Caption color={Colors.darkerGreen}>WIC</Caption>
+            </Chip>
+          )}
```
Comment (tommypoa) [inline components/store/StoreCard.js:141]: Can probably be a separate component since only the caption and font name is changed for similar components below!

Comment (tommypoa) [reply components/store/StoreCard.js:141]: Realised @wangannie also noted this in her review
